helm/flowforge/README.md

```diff
@@ -180,3 +174,12 @@ editors:
     create: true
     name: editors
 ```
+
+ ### Postgresql
+ - `postgresql.host` - the hostname of an external PostgreSQL database (default not set)
+ - `postgresql.port` - the port of an external PostgreSQL database (default `5432`)
+ - `postgresql.ssl` - sets the connection to the database to use SSL/TLS (default `false`)
+ - `postgresql.auth.username` - the username to use to connect to the database (default `forge`)
+ - `postgresql.auth.password` - the password to use to connect to the database (default `Zai1Wied`)
+ - `postgresql.auth.database` - the database to use (default `flowforge`)
+ - `postgresql.auth.postgresPassword` - the password to use for the postgres user (default `Moomiet0`)
```

Comment:
The defaults are no longer the case as they have been removed from the `values.yml`

Reply:
Ignore previous, they are still set